src/systems/a320_systems_wasm/src/lib.rs

```diff
@@ -162,6 +106,51 @@ async fn systems(mut gauge: msfs::Gauge) -> Result<(), Box<dyn Error>> {
             .provides_aircraft_variable("TURB ENG CORRECTED N2", "Percent", 2)?
             .provides_aircraft_variable("UNLIMITED FUEL", "Bool", 0)?
             .provides_aircraft_variable("VELOCITY WORLD Y", "feet per minute", 0)?
+            .with_aspect(|builder| {
+                builder.copy(
+                    Variable::aircraft("APU GENERATOR SWITCH", "Bool", 0),
+                    Variable::aspect("OVHD_ELEC_APU_GEN_PB_IS_ON"),
+                );
+                builder.copy(
+                    Variable::aircraft("BLEED AIR ENGINE", "Bool", 1),
+                    Variable::aspect("OVHD_PNEU_ENG_1_BLEED_PB_IS_AUTO"),
+                );
+                builder.copy(
+                    Variable::aircraft("BLEED AIR ENGINE", "Bool", 2),
+                    Variable::aspect("OVHD_PNEU_ENG_2_BLEED_PB_IS_AUTO"),
+                );
+                builder.copy(
+                    Variable::aircraft("EXTERNAL POWER AVAILABLE", "Bool", 1),
+                    Variable::aspect("OVHD_ELEC_EXT_PWR_PB_IS_AVAILABLE"),
+                );
+                builder.copy(
+                    Variable::aircraft("EXTERNAL POWER ON", "Bool", 1),
+                    Variable::aspect("OVHD_ELEC_EXT_PWR_PB_IS_ON"),
+                );
+
+                builder.copy(
+                    Variable::aircraft("GENERAL ENG MASTER ALTERNATOR", "Bool", 1),
+                    Variable::aspect("OVHD_ELEC_ENG_GEN_1_PB_IS_ON"),
+                );
+
+                builder.copy(
```

Comment:
Any reason there is an extra line break between some of these and not others?

Reply:
No. The line breaks were clearly confusing. I've now combined those items that are related. Should make more sense now 😀.